functions.php

```diff
@@ -20,7 +22,7 @@ function displayMouseOnPage(array $mice): string
 
 function linkDB(): PDO
 {
-    $db = new PDO('mysql:host=DB;dbname=collectionproject', 'root', 'password');
+    $db = new PDO('mysql:host=DB;dbname=collection.project', 'root', 'password');
```

Comment:
why?

Reply:
for some reason i couldnt access my table in my database so I made a new one and imported my table so it could work.

Reply:
i couldnt even access it in sequel pro